build: support local arm64 and amd64 image (#12754)
argoproj/argo-workflows: docs/running-locally.md

make argoexec-image
```

> Or you can use parameters to compile images that support specific platforms or multiple platforms with `TARGET_PLATFORM`:
> 

Reviewer: These are environment variables, not parameters
```suggestion
You can use the `TARGET_PLATFORM` environment variable to compile images for specific platforms:
```
- can also simplify this, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)

```bash

> # only support arm64
> make argoexec-image TARGET_PLATFORM=linux/arm64
> # only support amd64
> make argoexec-image TARGET_PLATFORM=linux/amd64
> # support arm64 and amd64

Reviewer: ```suggestion
# compile for both arm64 and amd64
```

- "compile", not "support"
- I think the last example is sufficient for all purposes, as it should be intuitive that you can remove the comma if you only need one, IMO. unless you think otherwise @Joibel

Author: yeah, thanks for your comment, i have fix it, could you have a look again ? @agilgur5

Reviewer: For future reference, you can use the "re-request a review" button on GH, then it also pops up in my notifications as a review request specifically
